src/main/java/sirius/biz/process/Processes.java

```diff
@@ -318,10 +318,7 @@ protected Process fetchStandbyProcess(String type, String tenantId) {
      * @return <tt>true</tt> if the process was successfully modified, <tt>false</tt> otherwise
      */
     private boolean modify(String processId, Predicate<Process> checker, Consumer<Process> modifier) {
-        Process process = process1stLevelCache.get(processId);
-        if (process == null) {
-            process = elastic.find(Process.class, processId).orElse(null);
-        }
+        Process process = fetchProcess(processId).orElse(null);
```

Comment:
i'd think we don't want to use the 2nd level cache here as it might contain a stale instance - we only use the 1st lvl cach to bypass the 1s write delay..